### PR TITLE
French translation - breaking changes section

### PR DESCRIPTION
**Pull request by Gimly**
## Description

This PR adds the French translation for the Breaking change section, with the breaking changes of the 2019 08 30. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* #6924

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by Gimly** [comment]
A bit smaller section, hope this isn't an issue, but it's simpler for me to manage the translation per section. All files of the section is translated, so it's ready for review, bring it on 😄.

**Comment by noroadsleft** [comment]
@zekth Please and thank you.

File: docs/fr-FR/breaking_changes.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+La période de breaking change est quand nous allons fusionner un PR qui change QMK d'une manière dangereuse ou inattendue. Il y a une période interne de test afin de nous assurer que les problèmes résiduels sont rares ou impossible à prévoir.
+
+## Qu'est-ce qui a été inclus dans des Breaking Changes précédents?
+
+* [30 août 2019](ChangeLog/20190830.md)
+
+## Quand va être le prochain Breaking Change?
+
+Le prochain Braking Change est planifié pour le 29 novembre.
```
Comment (Maxr1998) [inline docs/fr-FR/breaking_changes.md]:
```suggestion
Le prochain Breaking Change est planifié pour le 29 novembre.
```

File: docs/fr-FR/ChangeLog/20190830.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# QMK Breaking Change - 30 août 2019
+
+Quatre fois par an, QMK lance un processus pour fusionner les Breaking Changes. Un Breaking Change est un changement qui modifie la manière dont QMK fonctionne d'une manière incompatible ou un potentiellement dangereux. Nous n'effectuons ces changements que 4 fois par an afin que les utilisateurs n'aient pas peur de casser leurs keymaps en mettant à jour leur tree QMK.
```
Comment (zekth) [inline docs/fr-FR/ChangeLog/20190830.md]: `Un Breaking Change est un changement qui modifie la manière dont QMK fonctionne introduisant des incompatibilités ou des comportements dangereux.`
`en mettant à jour leur version de QMK.`

File: docs/fr-FR/ChangeLog/20190830.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+Ce document inaugure les fusions de Breaking Change. Voici la liste des changements.
```
Comment (zekth) [inline docs/fr-FR/ChangeLog/20190830.md]: `Ce document présente `

File: docs/fr-FR/ChangeLog/20190830.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+## Formattage de code Core avec clang-format
+
+* Tous les fichiers core (`drivers/`, `quantum/`, `tests/`, et `tmk_core/`) seront formattés avec clang-format
+* Un processus travis pour reformatter les PRs lors de la fusion a été mis en place
+* Vous pouvez utiliser la nouvelle commande CLI `qmk cformat` afin de formatter avant de soumettre votre PR si vous le souhaitez.
+
+## Nettoyage des descripteurs LUFA USB
+
+* Nettoyage du code lié aux descipteurs USB HID sur les claviers AVR, afin de les rendre plus simple à lire et compréhensibles
```
Comment (zekth) [inline docs/fr-FR/ChangeLog/20190830.md]: `descripteurs`

File: docs/fr-FR/ChangeLog/20190830.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+* Plus d'information:  https://github.com/qmk/qmk_firmware/pull/4871
+* Normalement pas de changement de fonctionnement et aucune keymap modifiée.
+
+## Migration des entrées de `ACTION_LAYER_MOMENTARY()` dans `fn_actions` vers des keycodes `MO()`
+
+* `fn_actions` est déprécié, et ses fonctionnalités ont été remplacées par des keycodes directs et `process_record_user()`
+* Supprimer cette fonctionnalité obsolète devrait résultat dans une réduction importante de la taille du firmware et de la complexité du code
```
Comment (zekth) [inline docs/fr-FR/ChangeLog/20190830.md]: `devrait aboutir à `

File: docs/fr-FR/ChangeLog/20190830.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+* Il est recommandé que toutes les keymaps affectées remplacent `fn_actions` vers les fonctionnalités de [keycode custom](https://docs.qmk.fm/#/custom_quantum_functions) et [macro](https://docs.qmk.fm/#/feature_macros)
+
+## Mis à jour Atreus vers les conventions de codage courantes
```
Comment (zekth) [inline docs/fr-FR/ChangeLog/20190830.md]: `Mise à jour`

File: docs/fr-FR/ChangeLog/20190830.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+* Les doublons include guards ont contourné le comportement de traitement d'en-tête prévu
```
Comment (zekth) [inline docs/fr-FR/ChangeLog/20190830.md]: `de traitements des headers attendu`

File: docs/fr-FR/ChangeLog/20190830.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+
+* Corrige une issue dans le fichier `keymap_br_abnt2.h` qui inclut la mauvaise souce (`keymap_common.h` au lieu de `keymap.h`)
+* Met à jour le fichier `keymap_swedish.h` afin d'être spécifique au suédois et plus "nordique" en général.
+* Toutes les keymaps qui utilisent ceci devront supprimer `NO_*` et le remplacer par `SE_*`.
+
+## Mise à jour du repo afin d'utiliser LUFA comme un sous-module git
+
+* `/lib/LUFA` supprimé du dépôt
+* LUFA, définis comme un sous-module, pointe vers qmk/lufa
+* Ceci devrait ajouter plus de flexibilité vers LUFA, et nous permet de garder le sous-module à jour bien plus facilement. Il avait environ 2 ans de retarder, sans manière simple de corriger. Ce changement devrait simplifier la mise à jour dans le futur.
```
Comment (zekth) [inline docs/fr-FR/ChangeLog/20190830.md]: `2 ans de retard`

File: docs/fr-FR/_summary.md [hunk 9]
```diff
@@ -0,0 +1,2 @@
+* [Changements de rupture](fr-FR/breaking_changes.md)
```
Comment (zekth) [inline docs/fr-FR/_summary.md]: `Breaking change` do not translate, otherwise it would become a mess.

File: docs/fr-FR/breaking_changes.md [hunk 10]
```diff
@@ -0,0 +1,107 @@
+# Breaking changes
+
+Ce document décrit le processus de QMK pour la gestion des breaking changes. Un breaking change est un changement qui modifie la manière dont QMK fonctionne d'une manière incompatible ou potentiellement dangereux. Nous limitons ces changements afin que les utilisateurs n'aient pas peur de casser leurs keymaps en mettant à jour leur tree QMK.
```
Comment (zekth) [inline docs/fr-FR/breaking_changes.md]: `fonctionne introduisant des incompatibilités ou des comportements dangereux.`
`en mettant à jour leur version de QMK.`

File: docs/fr-FR/breaking_changes.md [hunk 11]
```diff
@@ -0,0 +1,107 @@
...
+Le prochain Braking Change est planifié pour le 29 novembre.
+
+### Dates importantes
+
+* [x] 21 septembre 2019 - `future` est créé. Il va être rebasé de manière hebdomadaire.
+* [ ] 01 novembre 2019 - `future` fermé aux nouveaux PRs.
+* [ ] 01 novembre 2019 - Appel aux testeurs.
+* [ ] 27 novembre 2019 - `master` est bloqué, pas de PRs fusionnés.
+* [ ] 29 novembre 2019 - `future` est fusionné dans `master`.
+* [ ] 30 novembre 2019 - `master` est débloqul. Les PRs peuvent à nouveau être fusionnés.
```
Comment (zekth) [inline docs/fr-FR/breaking_changes.md]: `débloqué`

File: docs/fr-FR/breaking_changes.md [hunk 12]
```diff
@@ -0,0 +1,107 @@
...
+    * [ ] `git pull --ff-only`
+    * [ ] `git rebase origin/master`
+    * [ ] Modifie `readme.md`
+        * [ ] Supprimer les notes à propos de `future`
+    * [ ] Regroupe ChangeLog dans un fichier.
+    * [ ] `git commit -m 'Merge point for <DATE> Breaking Change'`
+    * [ ] `git push origin future`
+* Actions sur Github
+    * [ ] Crée un PR pour `future`
+    * [ ] S'assurer que Travis ne donne aucun problème
```
Comment (zekth) [inline docs/fr-FR/breaking_changes.md]: `ne relève`

File: docs/fr-FR/_summary.md (hunk 9 above)
Comment (Gimly) [reply docs/fr-FR/_summary.md]: I had written that initially and came back to the non translated version, forgot it in the summary

**Comment by noroadsleft** [comment]
@Gimly Can you rebase this branch and clean up the merge conflict please?

**Comment by Gimly** [comment]
@noroadsleft Sure, I'll rebase when you'll have merged #6925 since it will conflict again with this PR because it also changes the summary

**Comment by noroadsleft** [comment]
> @noroadsleft Sure, I'll rebase when you'll have merged #6925 since it will conflict again with this PR because it also changes the summary

You're way ahead of me, I was just thinking that was going to happen. 6925 is merged now; rebase when ready.

**Comment by Gimly** [comment]
@noroadsleft It's been rebased, all good for merge

**Comment by noroadsleft** [comment]
@Gimly As an aside, any future French translations should use `docs/fr-fr/` as the directory, rather than `docs/fr-FR/`. @jackhumbert and I discovered yesterday that the French docs don't actually work because the languages are configured for fully-lowercase paths, as Jack wants them. He has a commit to update the translation instructions, but it's not in master yet.

I have a branch in progress to fix the existing paths and docs, so don't worry about those. I'll add this PR's changes to it as well.

**Comment by Gimly** [comment]
@noroadsleft oops, sorry about the mix up with the capital letters, I'll fix it in the future translations I do, thanks for the heads up.

**Pull request merged**